draft-ietf-quic-transport.md

```diff
@@ -3847,13 +3879,41 @@ language from Section 3 of {{!TLS13=RFC8446}}.
 {: #figure-transport-parameters title="Definition of TransportParameters"}
 
 The `extension_data` field of the quic_transport_parameters extension defined in
-{{QUIC-TLS}} contains a TransportParameters value.  TLS encoding rules are
-therefore used to describe the encoding of transport parameters.
+{{QUIC-TLS}} contains a TransportParameters value.
 
 QUIC encodes transport parameters into a sequence of octets, which are then
 included in the cryptographic handshake.
 
 
+## Version Validation Fields {#transport-parameter-versions}
+
+Client and server exchange information about versions along with transport
```

Comment:
```suggestion
Client and server exchange information about versions in transport
```

Reply:
I have tried to make this separation clearer.  That is, there are transport parameters, and then there is the version negotiation info.  It makes talking about transport parameters easier.